README.md

```diff
@@ -162,3 +162,13 @@ This will use the included `tools/pydfu.py` script.  If flashing the firmware
 does not work it may be because you don't have the correct permissions, and
 need to use `sudo make deploy`.
 See the README.md file in the ports/stm32/ directory for further details.
+
+Contributing
+------------
+
+MicroPython is an open-source project and welcomes contributions. To be
+productive, please be sure to follow
+[Contributors' Guidelines](https://github.com/micropython/micropython/wiki/ContributorGuidelines)
+and [Code Conventions](https://github.com/micropython/micropython/blob/master/CODECONVENTIONS.md)
+docs. Note that MicroPython is licenced under the MIT license, and all
```

Comment:
Needs a few articles: "... follow the [Contributor's Guidelines] and the [Code Conventions]."

Also doesn't need the word "docs" because "Guidelines" and "Conventions" are already the noun.

Reply:
fixed